docs/css/main.css

```diff
@@ -325,6 +325,7 @@ a.anchorjs-link:hover { text-decoration: none; }
   border-bottom-width: 0px;
   margin-top: 0px;
   width: 210px;
+  height: 80%;
```

Comment:
Looks like it wouldn't work properly when the side bar is folded:

Before:

![Screen Shot 2020-11-12 at 1 24 49 PM](https://user-images.githubusercontent.com/6477701/98895470-88bd8100-24ea-11eb-80f0-7307a71ec3b5.png)


After:

![Screen Shot 2020-11-12 at 1 24 53 PM](https://user-images.githubusercontent.com/6477701/98895477-8c510800-24ea-11eb-92d1-534c865acbe4.png)

Reply:
also the end of bat does not seem matching:
![Screen Shot 2020-11-12 at 1 26 30 PM](https://user-images.githubusercontent.com/6477701/98895733-b9051f80-24ea-11eb-8aea-248a9053b580.png)